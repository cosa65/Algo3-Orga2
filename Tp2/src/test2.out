0.5
0.25
0
0.25
0
0.333333
0.333333
0.333333
