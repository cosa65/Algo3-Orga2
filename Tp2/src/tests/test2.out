0.5
0.366025
0
0.133975
0
0.633975
0.866025
0.866025
